package.json

```diff
@@ -34,25 +34,25 @@
   "devDependencies": {
     "@edge-runtime/jest-environment": "^2.3.3",
     "@edge-runtime/types": "^2.2.3",
+    "@types/express": "^5.0.0",
     "@types/jest": "^29.5.0",
-    "@types/node": "^18.11.17",
+    "@types/node": "^18.19.61",
     "@typescript-eslint/eslint-plugin": "^5.59.11",
     "@typescript-eslint/parser": "^5.59.11",
     "dotenv": "^16.0.3",
     "eslint": "^8.42.0",
     "eslint-plugin-import": "^2.27.5",
+    "express": "^4.21.1",
     "jest": "^29.5.0",
     "jest-progress-bar-reporter": "^1.0.25",
     "prettier": "^2.8.8",
     "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
     "typedoc": "^0.24.8",
-    "typescript": "^4.9.4"
+    "typescript": "^4.9.5"
   },
   "types": "dist/index.d.ts",
   "files": [
     "/dist"
-  ],
```

Comment:
Just some cleanup. This dep isn't needed anymore (it was moved to this section in [this PR](https://github.com/pinecone-io/pinecone-ts-client/commit/757185042c5c4703b5a4736f4e9836fcf1d65c6f) last year)